.gitignore

```diff
@@ -80,13 +80,10 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
```

Comment:
There is no reason to remove this. The gitignore is based on a template and since this isn't breaking anything in our project I see no reason to remove it.